client/checkout/classic/upe.js

```diff
@@ -12,7 +12,7 @@ import { getConfig, getCustomGatewayTitle } from 'utils/checkout';
 import WCPayAPI from '../api';
 import enqueueFraudScripts from 'fraud-scripts';
 import { getFontRulesFromPage, getAppearance } from '../upe-styles';
-import { getTerms } from '../utils/upe';
+import { getTerms, getCookieValue } from '../utils/upe';
```

Comment:
Do we still need this?

Reply:
I am using it to get the cart hash down here. Perhaps there's a better way to get the hash?

https://github.com/Automattic/woocommerce-payments/blob/5ec80784666a545b38bdd09062a81218007eae96/client/checkout/classic/upe.js#L668

Reply:
+1